activity5/obtain_cluster_information.py

```diff
@@ -0,0 +1,76 @@
+
+# coding: utf-8
+
+# In[1]:
+
+# load models
+import sys
+sys.path.append('../')
+from database.models import Incidencia, AccidentDBScan, DBSCANCluster, get_db_session, NumericCluster
+from sqlalchemy import func
+
+
+# In[2]:
+
+
+def obtain_characteristic(session, cluster, item, conditions):
+    ''' 
```

Comment:
I won't place the comment before the method because I follow PEP 257 for docstring comments (take a look at https://www.python.org/dev/peps/pep-0257/).